auth_token/config.py

```diff
@@ -15,6 +15,12 @@
     'MAX_TOKEN_AGE': 60 * 60 * 24 * 7 * 2,  # Max token expiration time (default: 2 weeks)
     'COUNT_USER_PRESERVED_TOKENS': 20,  # Maximum tokens that will be preserved per user
     'TAKEOVER_REDIRECT_URL': '/',  # The path where will be used after takeover another user token
+    'TWO_FACTOR_ENABLED': False,  # Two factor authentication is disabled
+    'TWO_FACTOR_REDIRECT_URL': 'login-code-verification/',  # The path the user is redirected to after successful two
+                                                            # factor authentication
+    'TWO_FACTOR_USER_CONTACT_FIELD_NAME': 'email',  # User field name which should be used for second part of two
```

Comment:
I don't see any use of this, except for checking it does exists, am I missing something?

Reply:
Maybe this is not needed at all, because anyway you need to implement your own sending function, which might not use this at all...

Reply:
I agree. This is not required